code/unclassified/leap_year/leap_year.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+using namespace std;
+
+bool checkLeapYear(int year)
+{
+    if (year % 400 == 0)
+        return true;
+
+    if (year % 100 == 0)
+        return false;
+
+    if (year % 4 == 0)
+        return true;
+    return false;
+}
+
+void findLeapYears(int a , int b)
+{
+    cout<<"The leap years between "<<a<<" and "<<b<<" are : \n";
```

Comment:
return a int set (i.e. vector<int>) for years, since this is a Utility for people who use it with other code.
in addition, you may add other function to find `first` leap year.

Reply:
Sorry but i didn't understood what u said . Plzz elaborate a little bit

Reply:
the function declare may look like this `std::vector<int> findLeapYears(int a , int b)`
and replace output of `cout` with `std::vector<int>`.